scripts/dev-server.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-env node */
+'use strict';
+
+const pkg = require('../package.json');
+const loadConfigFile = require('rollup/dist/loadConfigFile');
+const path = require('path');
+const rollup = require('rollup');
+const chokidar = require('chokidar');
+
+const { buildExamples } = require('./build-examples');
+
+const DEST_DIR = 'dist';
+const BROWSER_FILENAME = `${pkg.name}.js`;
+
```

Comment:
Wonder if we should flush the dist dir before starting this?

Reply:
Now that I'm thinking about it, I think it makes more sense to serve the dev build + examples from a different folder than `dist` which contains docs and the other build files.

Either way, I think flushing the output dir is a good call though.